app/views/admin/invoices/show.html.erb

```diff
@@ -1,6 +1,16 @@
-<h1>ADMIN <%= @invoice.id %></h1>
-<p>
-  Status: <%= @invoice.status %>
-  Created at: <%= @invoice.pretty_created_at %>
-  Customer: <%= @invoice.customer_name %>
-</p>
+<h1>ADMIN Invoice #<%= @invoice.id %></h1>
+  Status: <%= @invoice.status %><br>
+  Created at: <%= @invoice.pretty_created_at %><br>
+  Customer: <%= @invoice.customer_name %><br>
+  Items:
+  <hr>
+  <% @invoice.items_info.each_with_index do |info, index| %>
```

Comment:
Parker I liked how clean you were able to make this section with the each_with_index enumerable